plugin/screenshot/screenshot.py

```diff
@@ -26,13 +26,13 @@ class Actions:
     def screenshot(screen_number: Optional[int] = None):
         """Takes a screenshot of the entire screen and saves it to the pictures folder.
         Optional screen number can be given to use screen other than main."""
-        screen = get_screen(screen_number)
-        screenshot_rect(screen.rect)
+        selected_screen = get_screen(screen_number)
+        actions.user.screenshot_rect(selected_screen.rect)
 
     def screenshot_window():
         """Takes a screenshot of the active window and saves it to the pictures folder"""
         win = ui.active_window()
-        screenshot_rect(win.rect, win.app.name)
+        actions.user.screenshot_rect(win.rect, win.app.name)
```

Comment:
Need to put in a screen_num arg here (currently crashes).

Reply:
Easier to fix by adding keyword assignment `title=win.app.name`. I did that and confirmed it works.